site/src/pages/components/selectmenu.mdx

```diff
@@ -378,3 +379,12 @@ Like other form control elements, selectmenu supports validity checking attribut
 * [HTML spec for `checkValidity()`](https://html.spec.whatwg.org/#dom-cva-checkvalidity)
 * [HTML spec for `reportValidity()`](https://html.spec.whatwg.org/#dom-cva-reportvalidity)
 * [HTML spec for `setCustomValidity()`](https://html.spec.whatwg.org/#dom-cva-setcustomvalidity)
+
+## Why not improve `<select>`
+
+Instead of creating a new `<selectmenu>` element, we could improve the existing select element to meet the same needs. However, there are a few problems with this (borrowed from the [Custom Controls UI Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/ControlUICustomization/explainer.md)):
+
+- Changing the parsing behavior for `<select>` to allow other element children by default would likely break a lot of old sites that depend on the current behavior.
+- The `<select>` popup currently has the capability to break outside the window of the browser or of an iframe. Sites may depend on this behavior, but we would need to remove it for security reasons if `<select>` is allowed to contain arbitrary content.
```

Comment:
Akin to the previous bullet, we would need to modify this behavior also based on the existence of an opt-in attribute. We wouldn't be able to just remove the "break outside" capability for `<select>` across the board, that would not be web compatible.

I might phrase this slightly differently - I view this "break outside" capability as the super-power of the `<select>` element. When explaining to developers which element to choose, it's clearer if there are key differences in behavior. That story would go like this:
 - If you want full styleability and customizability, you need `<selectmenu>`. This element cannot, however, break outside the window boundary.
 - If you **need** your select picker to break outside the window boundary, you need `<select>`. This element cannot, however, be fully styled.

Reply:
This could be added after your last bullet below, instead.

Reply:
Ok, I rewrote this bullet point with your thoughts in mind. what do you think?